administrator/components/com_admin/sql/updates/postgresql/4.4.0-2023-05-08.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder", "client_id", "enabled", "access", "protected", "locked", "manifest_cache", "params", "custom_data", "ordering", "state") VALUES
+	(0, "plg_quickicon_eos", "plugin", "eos", "quickicon", 0, 1, 1, 0, 1, "", "", "", 7, 0);
```

Comment:
```suggestion
	(0, 'plg_quickicon_eos', 'plugin', 'eos', 'quickicon', 0, 1, 1, 0, 1, '', '', '', 7, 0);
```
Only name quotes are double quotes in PostgreSQL, string quotes are single quotes like in MySQL.

Reply:
P.S.: That's not just code style, it's a syntax error.

Reply:
Thanks. Not worked with Postgres for about 25 years.